pkg/services/ngalert/api/api_prometheus.go

```diff
@@ -71,9 +71,14 @@ func (srv PrometheusSrv) RouteGetRuleStatuses(c *models.ReqContext) response.Res
 		namespaceUIDs = append(namespaceUIDs, k)
 	}
 
+	dashboardUID := c.Query("dashboard_uid")
+	panelID := c.QueryInt64("panel_id")
```

Comment:
I think I need add validation here as `QueryInt64` returns `0` if `panel_id` does not contain a valid `int64`?